### ISSUE TITLE
Add a way to express proxy connections

### ISSUE THREAD

**Issue by Skyr** [Threagile/threagile#15]
It would be great to show communication links passing through proxies - e.g. by adding a "via" field (or even list) to the communication_links entries.

**Comment by cschneider4711** [comment]
Yep, that's something useful for reverse proxies or stuff like that... 

There's already something like that in the queue to have a technical component act like a transparent element in risk analysis. Ideas exist for being able to have "n" incoming communication links to a reverse-proxy (or WAF) like thing and then "m" connections outgoing but still being able to address what input leads to what specific output connection. 

Current state is: haven't fully thought through about how to add this seamless and intuitively (for the user) to the model, but will post some ideas soon (currently prepping a conference talk about Threagile) and look forward for feedback then.
